Resources/Textures/Structures/Machines/techfab.rsi/meta.json

```diff
@@ -13,6 +13,9 @@
     {
       "name": "med"
     },
+    {
+        "name": "ammo"
```

Comment:
did you make this sprite? if so add yourself to the copyright (something like `X made by YOU for SS14`)